Spring-Project/src/main/java/com/epam/project/spring/dto/UserDto.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.project.spring.dto;
+
+import com.epam.project.spring.model.UserRole;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class UserDto {
+
+    private String firstName;
+    private String lastName;
+    private String email;
+    private UserRole role;
```

Comment:
Hi Zenyk. Please introduce **UserRoleDto** or use **String** instead of **UserRole** model

Reply:
Thanks. Resolved